libs/core/package.json

```diff
@@ -19,13 +19,13 @@
   "license": "MIT",
   "peerDependencies": {
     "@angular/common": "^9.0.0",
-    "@angular/core": "^9.0.0",
-    "tslib": "^1.0.0"
+    "@angular/core": "^9.0.0"
```

Comment:
If it is migrating to Angular 10, why it is still `@angular/core` version 9 ?

Reply:
I have these versions updated locally, just did not commit yet as I'm trying to figure out the errors in Travis CI...

Reply:
Package updated to v10.